Buildings/Fluid/Storage/Plant/ChillerAndTank.mo

```diff
@@ -0,0 +1,418 @@
+    TConEntMax=333.15)
+    "Chiller performance data"
+    annotation (choicesAllMatching=true,
+      Placement(transformation(extent={{-100,82},{-80,102}})));
+
+  Buildings.Fluid.FixedResistances.PressureDrop preDro2(
+    redeclare package Medium = Medium2,
+    final allowFlowReversal=true,
+    final dp_nominal=dp_nominal/10,
+    final m_flow_nominal=mTan_flow_nominal) "Flow resistance on tank branch"
+    annotation (Placement(transformation(extent={{10,-10},{-10,10}},
+        rotation=-90,
+        origin={-80,-80})));
+  Modelica.Blocks.Interfaces.RealInput set_mPum1_flow
+    "Primary pump mass flow rate setpoint" annotation (Placement(transformation(
+        extent={{-10,-10},{10,10}},
+        rotation=-90,
+        origin={40,130}),  iconTransformation(
+        extent={{-10,-10},{10,10}},
+        rotation=0,
+        origin={-110,100})));
+
+  Buildings.Fluid.Movers.FlowControlled_m_flow pum1(
+    redeclare package Medium = Medium2,
+    per(pressure(dp=dp_nominal*{2,1.2,0}, V_flow=mChi_flow_nominal/1.2*{0,1.2,2})),
+    energyDynamics=Modelica.Fluid.Types.Dynamics.FixedInitial,
+    allowFlowReversal=true,
+    addPowerToMedium=false,
+    m_flow_nominal=mChi_flow_nominal,
+    m_flow_start=0,
+    T_start=T_CHWR_nominal) "Primary CHW pump"
+    annotation (Placement(transformation(extent={{-10,-42},{-30,-22}})));
+
+  Modelica.Blocks.Interfaces.RealOutput mTan_flow
+    "Mass flow rate through the tank" annotation (Dialog(group=
+          "Time varying output signal"), Placement(transformation(
+        extent={{10,-10},{-10,10}},
+        rotation=90,
+        origin={0,-110}),   iconTransformation(
+        extent={{10,-10},{-10,10}},
+        rotation=180,
+        origin={110,-20})));
+  Buildings.Fluid.Movers.SpeedControlled_y pum2(
+    redeclare package Medium = Medium2,
+    per(pressure(dp=dp_nominal*{2,1.2,0}, V_flow=(mChi_flow_nominal +
+            mTan_flow_nominal)/1.2*{0,1.2,2})),
+    energyDynamics=Modelica.Fluid.Types.Dynamics.FixedInitial,
+    allowFlowReversal=true,
+    addPowerToMedium=false,
+    y_start=0,
+    T_start=T_CHWR_nominal) "Secondary pump" annotation (Placement(transformation(
+        extent={{10,-10},{-10,10}},
+        rotation=0,
+        origin={80,-20})));
+
+  Buildings.Fluid.Storage.Plant.BaseClasses.ReversiblePumpValveControl conPum2Gro
+    if allowRemoteCharging "Control block for secondary pump-valve group"
+    annotation (Placement(transformation(extent={{80,80},{60,102}})));
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanInput booOnOff
+    if allowRemoteCharging "Plant status: true = on; false = off"
+                                                   annotation (Placement(
+        transformation(extent={{120,70},{100,90}}),
+        iconTransformation(extent={{-140,-120},{-100,-80}})));
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanInput booFloDir
+    if allowRemoteCharging
+    "Flow direction: true = normal; false = reverse" annotation (Placement(
```

Comment:
See comment in controller and update naming here too.

Reply:
Renamed to `uRemCha` in 3b78e69.